logging/unit_tests/test_client.py

```diff
@@ -375,25 +377,42 @@ def test_list_sinks_no_paging(self):
             'filter': FILTER,
             'destination': self.DESTINATION_URI,
         }]
-        client = self._makeOne(project=PROJECT, credentials=_Credentials())
-        api = client._sinks_api = _DummySinksAPI()
-        api._list_sinks_response = SINKS, TOKEN
+        client = self._makeOne(project=PROJECT, credentials=_Credentials(),
+                               use_gax=False)
+        returned = {
+            'sinks': SINKS,
+            'nextPageToken': TOKEN,
+        }
+        client.connection = _Connection(returned)
 
-        sinks, token = client.list_sinks()
+        iterator = client.list_sinks()
+        page = six.next(iterator.pages)
+        sinks = list(page)
+        token = iterator.next_page_token
```

Comment:
Sure. Not sure what the value is that the two existing tests don't already provide?